{
  try {
    fooBoxTest();
    fail();
  }
 catch (  JavaScriptException e) {
  }
}
